src/Coduo/TuTu/Config/Element.php

```diff
@@ -76,10 +76,8 @@ private static function createArrayConfigResolver()
         $configResolver = new OptionsResolver();
         $configResolver->setRequired(['request']);
         $configResolver->setDefaults(['response' => []]);
-        $configResolver->setAllowedTypes([
-            'request' => 'array',
-            'response' => 'array',
-        ]);
+        $configResolver->setAllowedTypes('request', 'array');
+        $configResolver->setAllowedTypes('response', 'array');
```

Comment:
The interface [changed slightly](http://api.symfony.com/3.2/Symfony/Component/OptionsResolver/OptionsResolver.html#method_setAllowedTypes). This was the reason why specs failed.